people/good-newcomers.md

```diff
@@ -1,12 +1,12 @@
 # Newcomers can be awesome
 
-There are lots of clever people in the world. A person who never previously
-contributed before in your project can be really bright, smart and write
+There are lots of clever people in the world. People who never previously
+contributed to your project can be really bright, smart and write
 incredible code that pushes your team forward already in their first
 contributions. When responding to questions and providing review comments on
-changes, remember that new in the project does not necessarily imply rookie in
+changes, remember that being new in the project does not necessarily imply rookie in
```

Comment:
Hi, Gábor!

I agree that adding "being" makes this read better, and to complete the change, it would also need an addition like "...does not necessarily imply *that someone is a* rookie".

Reply:
added